content/client-server-api/modules/content_repo.md

```diff
@@ -49,8 +49,8 @@ mxc://<server-name>/<media-id>
 Clients can access the content repository using the following endpoints.
 
 {{% boxes/added-in-paragraph %}}
-{{< changed-in v="1.11" >}} Clients SHOULD NOT use the deprecated media endpoints
-described below. Instead, they SHOULD use the new endpoints which require authentication.
+{{< changed-in v="1.11" >}} Clients SHOULD NOT use the media endpoints marked as
+deprecated below. Instead, they SHOULD use the new endpoints which require authentication.
```

Comment:
To echo what was already said in the SCT room: This seems like an improvement over the previous text but it still mainly just rehashes what "deprecated" means. @richvdh's suggestion was something along the lines of:

> "[Changed in v1.11] A number of endpoints under the /_matrix/media hierarchy have been deprecated and replaced with new endpoints which require authentication"

Reply:
Thanks, I've adjusted the text based on this feedback